lib/screens/home/components/recomend_products.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+
+import '../../../constants.dart';
+
+class RecomendsProducts extends StatelessWidget {
+  const RecomendsProducts({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      scrollDirection: Axis.horizontal,
+      child: Row(
+        children: <Widget>[
+          RecomendProductCard(
+            image: "assets/images/image_1.PNG",
+            title: "Tesla S",
+            city: "Львів",
+            price: 24400,
+            press: () {
+            },
+          ),
+          RecomendProductCard(
+            image: "assets/images/image_2.PNG",
+            title: "Iphone 11 pro",
+            city: "Холодновідка",
+            price: 999,
+            press: () {
+
+            },
+          ),
+          RecomendProductCard(
+            image: "assets/images/image_3.PNG",
+            title: "imac 24' 2021",
+            city: "скнилів",
+            price: 2500,
+            press: () {},
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class RecomendProductCard extends StatelessWidget {
+  const RecomendProductCard({
+    Key key,
+    this.image,
+    this.title,
+    this.city,
+    this.price,
+    this.press,
+  }) : super(key: key);
+
+  final int price;
+  final String image, title, city;
+  final Function press;
+
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    return Container(
+      margin: EdgeInsets.only(
+        left: kDefaultPadding,
+        top: kDefaultPadding / 2,
+        bottom: kDefaultPadding * 2.5,
```

Comment:
not sure where you all are taking this `k` prefix from, but I would get rid of it